invoice2.dtx

```diff
@@ -58,7 +58,7 @@
 % snippets designed for creating invoices, the only solution available in the
 % CTAN is the \pkg{invoice}\footnote{\url{https://www.ctan.org/pkg/invoice}}
 % package, which is not actively maintained and lacks
-% e.\,g.~internationalization support while requiring use of a project
+% e.g.~internationalization support while requiring use of a project
```

Comment:
I decided to eliminate the non-breaking spaces in the abbreviations since "Don't use any sort of space between the letters and the periods of an abbreviation" is widely consensus of English typography. I recompiled the document to see if that change led to awkward linebreaks but that wasn't the case on my setup.